pkg/scheduler/scheduler.go

```diff
@@ -240,7 +240,10 @@ func (s *Scheduler) nominate(ctx context.Context, workloads []workload.Info, sna
 		cq := snap.ClusterQueues[w.ClusterQueue]
 		ns := corev1.Namespace{}
 		e := entry{Info: w}
-		if snap.InactiveClusterQueueSets.Has(w.ClusterQueue) {
+		if s.cache.IsAssumedOrAdmittedWorkload(w) {
+			log.Error(fmt.Errorf("workload is assumed"), "nominate", "workloadInfo", w)
```

Comment:
This is not an error
```suggestion
			log.Info("Workload skipped from admission because it's already assumed or admitted"), "workload", klog.KObj(w.Obj))
```